src/app/qgsidentifyresultsdialog.cpp

```diff
@@ -618,14 +618,17 @@ void QgsIdentifyResultsDialog::addFeature( QgsVectorLayer *vlayer, const QgsFeat
     if ( foundLinks )
     {
       QLabel *valueLabel = new QLabel( links );
+      valueLabel->setWordWrap( true );
       valueLabel->setOpenExternalLinks( true );
       attrItem->setData( 1, Qt::DisplayRole, QString() );
       attrItem->treeWidget()->setItemWidget( attrItem, 1, valueLabel );
     }
     else
     {
-      attrItem->setData( 1, Qt::DisplayRole, value );
-      attrItem->treeWidget()->setItemWidget( attrItem, 1, nullptr );
+      auto valueLabel { qgis::make_unique<QLabel>( value ) };
```

Comment:
This looks unrelated?

Reply:
What do you mean? That a unique pointer is not required here and a plain `new` would do just fine?

Reply:
@nyalldawson gentle ping

Reply:
I mean I can't see what it's got to do with snapping settings...?

Reply:
Oh sorry, I see it's a leftover from this https://github.com/qgis/QGIS/issues/31650#issuecomment-535090459  , I'll remove it.